lib/binding_rust/lib.rs

```diff
@@ -113,7 +113,7 @@ pub struct TreeCursor<'cursor>(ffi::TSTreeCursor, PhantomData<&'cursor ()>);
 
 /// A set of patterns that match nodes in a syntax tree.
 #[doc(alias = "TSQuery")]
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
The `TSQuery` is not reference counted internally, so I don't think this derived `Clone` implementation will work. Queries are also immutable, so I don't think we should need to make them clone-able.

Reply:
For the same reason I [removed](https://github.com/tree-sitter/tree-sitter/blob/016d8c24998cc268d6a6c5da471f2cc11a1c0573/script/generate-bindings#L12) [this](https://github.com/tree-sitter/tree-sitter/blob/016d8c24998cc268d6a6c5da471f2cc11a1c0573/lib/binding_rust/build.rs#L53) also from bindings generation.